service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksStep.java

```diff
@@ -122,6 +69,148 @@ protected void createResource(FlightContext context, ArmManagers armManagers) {
     logger.info(RESOURCE_CREATED, getResourceType(), aks.id(), getMRGName(context));
   }
 
+  private KubernetesCluster createAks(FlightContext context) {
+    UUID landingZoneId =
+        getParameterOrThrow(
+            context.getInputParameters(), LandingZoneFlightMapKeys.LANDING_ZONE_ID, UUID.class);
+    var vNetId = getParameterOrThrow(context.getWorkingMap(), CreateVnetStep.VNET_ID, String.class);
+
+    var aksName = resourceNameProvider.getName(getResourceType());
+
+    KubernetesCluster aks = null;
+    //    try {
```

Comment:
Will want to delete the commented-out code.

Reply:
Interesting.. maybe some Github artifact after rebasing. It was removed.